src/java/org/apache/cassandra/db/repair/CassandraValidationIterator.java

```diff
@@ -270,7 +272,7 @@ public void close()
         {
             // we can only clear the snapshot if we are not doing a global snapshot validation (we then clear it once anticompaction
             // is done).
-            cfs.clearSnapshot(snapshotName);
+            SnapshotManager.instance.clearSnapshot(cfs.keyspace.getName(), cfs.name, snapshotName);
```

Comment:
for consistency with the other changes in this file:
```suggestion
            SnapshotManager.instance.clearSnapshot(cfs.getKeyspaceName(), cfs.getTableName(), snapshotName);
```

Reply:
OK